types/run.sh

```diff
@@ -68,18 +68,31 @@ typed-arith-asdl() {
   echo
 }
 
-typecheck-more-oil() {
-  #typecheck $flags osh/word_compile.py
 
-  local log=_tmp/typecheck-more-oil.txt
+readonly MORE_OIL_MANIFEST=types/more-oil-manifest.txt
+
 
-  set +o errexit
-  typecheck $MYPY_FLAGS \
-    osh/glob_.py osh/string_ops.py frontend/location.py \
-    osh/history.py core/comp_ui.py \
-    > $log
+need-typechecking() {
+    # This command is useful to find files to annotate and add to
+    # $MORE_OIL_MANIFEST.
+    # It shows all the files that are not included in
+    # $MORE_OIL_MANIFEST or $OSH_PARSE_MANIFEST, and thus are not yet
+    # typechecked by typecheck-more-oil here or
+    # `types/osh_parse.sh travis`.
+    comm -2 -3 \
+         <(metrics/source-code.sh osh-files | grep '.py$' | sed 's@^@./@') \
```

Comment:
Doesn't need to be changed for this commit, but when doing my own audits like this, I noticed we were leaving out `oil_lang/` (and there is no `oil-files` -- it's a bit approximate)

For example `oil_lang/regex_translate.py` is the Eggex to POSIX ERE translator that's probably very easy to type check (and translate), since it's all string manipulation.